### ISSUE TITLE
Exporting git logs incorrect

### ISSUE THREAD

**Issue by josinaldobarbosa** [bitrise-steplib/steps-git-clone#101]
I am using Authenticate host with netrc to configure my git, in some cases the information is not exported correctly, after adding the user and email as recommended, GIT_CLONE_COMMIT_HASH remains the default ...

```
HEAD is now at 74c3eb2... Merge commit '9ba8c2e71d05a03f6d222edb71b8d1276defa718' into develop
Exporting git logs
=> GIT_CLONE_COMMIT_AUTHOR_EMAIL
   value: please-set-your-email@bitrise.io
=> GIT_CLONE_COMMIT_COMMITER_NAME
   value: J. Doe (https://devcenter.bitrise.io/builds/setting-your-git-credentials-on-build-machines/)
=> GIT_CLONE_COMMIT_COMMITER_EMAIL
   value: please-set-your-email@bitrise.io
=> GIT_CLONE_COMMIT_HASH
   value: 74c3eb2292f3070a72e65761e01f7ec90bcd582a
=> GIT_CLONE_COMMIT_MESSAGE_SUBJECT
   value: Merge commit '9ba8c2e71d05a03f6d222edb71b8d1276defa718' into develop
=> GIT_CLONE_COMMIT_MESSAGE_BODY
   value: 
=> GIT_CLONE_COMMIT_AUTHOR_NAME
   value: J. Doe (https://devcenter.bitrise.io/builds/setting-your-git-credentials-on-build-machines/)
=> GIT_CLONE_COMMIT_COUNT
   value: 3332
```
Example build: https://app.bitrise.io/build/cfb06cc3ead86397

**Comment by bitce** [comment]
Hi @josinaldobarbosa!

Sorry to hear about this problem, can you please also include an URL for a build where this is happening?

**Comment by josinaldobarbosa** [comment]
@bitce added

**Comment by josinaldobarbosa** [comment]
@bitce added

**Comment by godrei** [comment]
Hi @josinaldobarbosa , this issue seems to be a duplicate of https://github.com/bitrise-steplib/steps-git-clone/issues/89.

The source of the issue is the same: while the step checks out a PR, based on the PR checkout strategy it might call git merge, which can result in a new commit. The step exports the outputs once the desired state's checkout is done, that's why the output can be different than the PR commit's info.

I will close this issue and share the status of the resolution on the linked issue.
